### PR TITLE
Update waas-restart.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more registry details to the document

Fixes https://github.com/MicrosoftDocs/windows-docs-pr/issues/9347

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/deployment/update/waas-restart.md [hunk 1]
```diff
@@ -199,10 +199,15 @@ The following tables list registry values that correspond to the Group Policy se
...
+| NoAutoUpdate | REG_DWORD | 0: Automatic Updates is enabled (default)</br>1: Automatic Updates is disabled |
 | AlwaysAutoRebootAtScheduledTimeMinutes | REG_DWORD | 15-180: set automatic reboot to occur after given minutes |
 | AUOptions | REG_DWORD | 2: notify for download and notify for installation of updates</br>3: automatically download and notify for installation of updates</br>4: Automatically download and schedule installation of updates</br>5: allow the local admin to configure these settings</br>**Note:** To configure restart behavior, set this value to **4** |
 | NoAutoRebootWithLoggedOnUsers | REG_DWORD | 0: disable do not reboot if users are logged on</br>1: do not reboot after an update installation if a user is logged on</br>**Note:** If disabled: Automatic Updates will notify the user that the computer will automatically restart in 5 minutes to complete the installation  |
+| UseWUServer | REG_DWORD | Set this value to 1 to configure Automatic Updates to use a server that is running Software Update Services instead of Windows Update |
+| ScheduledInstallDay |  | 0: Every day</br>1 through 7: The days of the week from Sunday (1) to Saturday (7).
```
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-restart.md:207]:
```suggestion
| ScheduledInstallDay |  | 0: every day</br>1 through 7: the days of the week from Sunday (1) to Saturday (7)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mestew Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mestew Gentle reminder. Thanks

**Comment by mestew** [comment]
declining change since specific article is tightly scoped to restart settings and while an argument can be made for some of the suggested reg values, they are slightly outside the scope

**Pull request closed without merging**